composer.json

```diff
@@ -9,25 +9,29 @@
         "xmlschema"
     ],
     "license" : "MIT",
-    "authors" : [{
+    "authors" : [
+        {
             "name" : "Asmir Mustafic",
             "email" : "goetas@lignano.it"
         }
     ],
+    "config": {
+        "sort-packages": true
+    },
     "require" : {
-        "php" : ">=5.4.0"
+        "php": ">=5.6"
     },
     "require-dev" : {
-        "phpunit/phpunit" : "^4.8|^5.0"
+        "phpunit/phpunit": "^5.7"
     },
     "autoload" : {
         "psr-4" : {
+            "GoetasWebservices\\CS\\": "./php-cs-fixer/",
```

Comment:
why is this in the `autoload` section instead of `autoload-dev` ?

Reply:
This was either an error on my part, or a means of ensuring the composer.json files of downstream projects don't need their autoloaders modified in order to load up the config [as in php_cs.dist on this repo](https://github.com/goetas-webservices/xsd-reader/blob/bb953a590c79e0f8e800e913a50768a40ed9daa1/.php_cs.dist)